slave-nodejs6-centos7/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM fhwendy/jenkins-slave-base-centos7
+
+MAINTAINER Gerard Ryan <gryan@redhat.com>
+
+ENV ENABLED_COLLECTIONS="${ENABLED_COLLECTIONS} rh-nodejs6" \
+    PATH="/home/jenkins/bin:${PATH}"
+
+USER root
+
+RUN INSTALL_PKGS="rh-nodejs6 rh-nodejs6-npm memcached bzip2 jq fontconfig" && \
+    yum install -y epel-release && \
+    yum install -y $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    yum clean all && \
+    rm -rf /var/cache/yum && \
+    chmod -R 770 ${HOME}
+
+USER 1001
+
+RUN echo "prefix=${HOME}" > ${HOME}/.npmrc && \
+    scl enable rh-nodejs6 "npm i -g grunt jira-miner" && \
+    curl -o ${HOME}/bin/openshift-template-tool https://github.com/feedhenry/openshift-template-tool/releases/download/0.0.2/openshift-template-tool-linux-amd64 && \
+    chmod 755 ${HOME}/bin/openshift-template-tool && \
+    chmod -R 775 ${HOME}/lib ${HOME}/etc ${HOME}/bin ${HOME}/.npm ${HOME}/.npmrc
```

Comment:
Do you not need to call that configure-slave script somewhere so it's sources when the container starts?https://github.com/feedhenry/wendy-jenkins-s2i-configuration/blob/master/slave-ruby-fhcap/Dockerfile#L7-L9

Reply:
Probably a good idea to add a test for this new Dockerfile so we can check nodejs is loaded and the correct version https://github.com/feedhenry/wendy-jenkins-s2i-configuration/blob/master/slave-ruby/test/run

Reply:
With regards to calling the configure-slave script, we don't need to do that explicitly, since it's already sourced by the `run-jnlp-client` script here: https://github.com/feedhenry/wendy-jenkins-s2i-configuration/blob/master/slave-base-centos7/contrib/bin/run-jnlp-client#L37

I'll add a test similar to the other ones. 👍

Reply:
Cool, just couldn't see where that was happening.

Reply:
@mikenairn I've added a test script to the most recent push here.